app/models/budget.rb

```diff
@@ -17,6 +15,15 @@ def pct_remaining
     end
   end
 
+  def downcase_category
```

Comment:
yep, this looks like what we were talking about yesterday. all categories should be downcased before being saved